### PR TITLE
New brand portals

### PR DESCRIPTION
**Pull request by mmadariaga**
Created new brand portal SPA

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/portal/brand/src/entities/ActiveCalls/ActiveCalls.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import CellTowerIcon from '@mui/icons-material/CellTower';
+import EntityInterface, {
+  OrderDirection,
+} from '@irontec/ivoz-ui/entities/EntityInterface';
+import DefaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import { BillableCallProperties } from './ActiveCallsProperties';
```
Comment (Kaian) [inline web/portal/brand/src/entities/ActiveCalls/ActiveCalls.tsx:7]: BillableCallProperties -> ActiveCallsProperties

**Review by Kaian — CHANGES_REQUESTED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/brand/src/entities/Company/Form.tsx [hunk 2]
```diff
@@ -16,43 +16,86 @@ const Form = (props: EntityFormProps): JSX.Element => {
     match,
   });
 
+  const type = row?.type ?? '';
+  const isVpbx = type === 'vpbx';
+  const isRetail = type === 'retail';
+  console.log('isRetail', isRetail);
```
Comment (Kaian) [inline web/portal/brand/src/entities/Company/Form.tsx:22]: debugging lines here

File: web/portal/brand/src/entities/Company/Company.tsx [hunk 3]
```diff
@@ -185,7 +210,7 @@ const properties: CompanyProperties = {
     },
   },
   balance: {
-    label: _('Balance'),
+    label: _('Balance'), //WTF
```
Comment (Kaian) [inline web/portal/brand/src/entities/Company/Company.tsx:213]: WTF do you mean with WTF!

File: web/portal/brand/src/entities/Company/Form.tsx [hunk 4]
```diff
@@ -16,43 +16,86 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+  const isResidential = type === 'residential';
+  //const isWholesale = type === 'wholesale';
```
Comment (Kaian) [inline web/portal/brand/src/entities/Company/Form.tsx:24]: Commented code

**Comment by Kaian** [comment]
I have added 2 commits here:
 - DDI description field from #1926 
 - Apache configuration to access this portal through /brand

Keep up the good work! :muscle:

**Review by Kaian — APPROVED** [review]

**Pull request merged**